packages/google-drive-kit/tests/utils.ts

```diff
@@ -0,0 +1,18 @@
+import test from "ava";
```

Comment:
recommend using node built-in test framework. An example of such tests is here:

https://github.com/breadboard-ai/breadboard/tree/main/packages/breadboard/tests/node

and the configuration is here:

https://github.com/breadboard-ai/breadboard/blob/main/packages/breadboard/package.json#L73

since there aren't any other kinds of tests in this package, no need to put them into tests/node -- just tests should work